src/app/main.rs

```diff
@@ -728,9 +733,9 @@ where
             let source_stack = dst_router
                 .push(orig_proto_downgrade::layer())
                 .push(insert_target::layer())
-                .push(set_remote_ip_on_req::layer())
+                //.push(set_remote_ip_on_req::layer())
+                //.push(set_client_id_on_req::layer())
```

Comment:
Is it necessary to no longer set inbound client IDs? AFAICT that should _not_ make it out of the cluster even when injected into an ingress controller?

Reply:
I don't think there's any value in leaving this stuff half-on.

Reply:
Fair enough!